auth/tenant_mgt.go

```diff
@@ -275,6 +282,11 @@ func (t *TenantToUpdate) EnableEmailLinkSignIn(enable bool) *TenantToUpdate {
 	return t.set(enableEmailLinkSignInKey, enable)
 }
 
+// EnableAnonymousUsers enables or disables anonymous user.
```

Comment:
Our auth docs use the term "anonymous authentication." Should that be used here?

I'd suggest that phrase, or, alternatively, "anonymous user access."

Reply:
This makes sense, but in Go SDK we normally mention the function name followed by a description. Ex: `AllowPasswordSignUp enables or disables email sign-in provider.` or `EnableEmailLinkSignIn enables or disables email link sign-in.`

Reply:
Hm, OK, no backticks I could live with.  But just "anonymous user" doesn't sound to me like a thing that can be enabled or disabled.  Does it to you Lahiru?

I'd say that "email link sign-in" *does* sound like it can be disabled/enabled.  FWIW at this point, "email sign-in provider" does not; "email sign-in provider support" or "email sign-in provider functionality" might.

This sounds like I'm splitting hairs, but one of the few things I bring to the table is an ear for written English.  :)  Let's get Kevin's input as well!

Reply:
Agreed.

"EnableAnonymousUsers enables or disables anonymous authentication."

(And FWIW, "AllowPasswordSignUp enables or disables the email sign-in provider.")

Reply:
+1  Yup, "the" solves that latter one!

Reply:
Oh I misread your previous comment, sorry! I understand what we are referring to now.
I am good with `// EnableAnonymousUsers enables or disables anonymous user authentication.`
Thank you!